### PR TITLE
Translations through i18next

### PR DESCRIPTION
**Pull request by YeetHackz**
Would like to make changes to README to ask for help translating. I will be adding more changes for full Localization support, although I need the Translations. If you have any ideas or languages, for this feature, in mind, please drop them below. Thank you!

### CODE REVIEW

**Comment by YeetHackz** [comment]
Hello World!
First, I would like to thank @lexxndr for the support translating. I would love to see Vulx translated and localized.  
Second, they have raised a question: Should "lockfile" be translated? or not because it is the file?
Thank you all for you support. I look forward to seeing more community input and translations.

**Comment by ItsSyfe** [comment]
> Hello World! First, I would like to thank @lexxndr for the support translating. I would love to see Vulx translated and localized. Second, they have raised a question: Should "lockfile" be translated? or not because it is the file? Thank you all for you support. I look forward to seeing more community input and translations.

We're still unsure about if we want to translate Vulx, if someone wants to implement a good solution for it and we're happy with it then we will merge it. As for your other question, lockfile is where we grab the details Valorant uses for interacting with their local client api and as such you shouldn't interfere with the pre-existing code for it there.

**Comment by YeetHackz** [comment]
Hi Syfe. I guess I should explain in more detail what I am / am planning to do. 
I have already setup a Crowdin Page. This allows the community to supply translations for the strings.
I am in the process of adding the text strings used in Vulx to a language file and uploading to Crowdin for translation. 
Once I have completed this and have some community contributed translations, I plan to improve the (although existing, basic) translation code. 
I hope to have a UI option to select a language (currently grabs the system language). 

TL;DR: I hope to develop a translation and localization solution for Vulx that can be easily pulled, merged and maintained. Having your application translated to user's native languages will help expand your audience and user experience. Thank you!

**Comment by YeetHackz** [comment]
Sorry. As for the lockfile, we were not planning to modify the code. The question about the lockfile related to translation of the UI and logs. We were wondering if the word: "lockfile" in the logs and UI should be translated or left as "lockfile". Hope this clarification helps. Sorry I missed this earlier.

**Comment by ItsSyfe** [comment]
That's fine by us, just as for translating only the GUI text needs to be translated you should not translate anything being logged by the backend as that creates more hassle fixing issues and most of those logs are prone to bring changed. Thanks!

**Comment by YeetHackz** [comment]
Awesome! So, we should exclude all strings from the logger? Or just anything that isn't in a .html? 
Thanks. Keep up the amazing work. <3

**Comment by AquaPlaysYT** [comment]
Keep in mind #36 is in progress to allow translations. If possible work with @NeXiDE to implement this and create a pull together.
You can add him on [Discord](discord.com/users/252451046889095168).

**Review by AquaPlaysYT — APPROVED** [review]
All good, thank you for creating a pull, I recommend you work with Nexi on this. Im unsure if you can add him directly do the pull but if you can go ahead.

**Comment by YeetHackz** [comment]
Awesome! Thanks for your help Syfe. I will continue to push the locale files as they are translated by the community. Thank you everyone on the Vulx team.

File: public/js/vulx.localization.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ $(function () {
 	  .init({
 		debug: true,
 		fallbackLng: 'en',
-		ns: ["default"],
-		defaultNS: "default",
+		ns: ["en-AU"],
```
Comment (ItsSyfe) [inline public/js/vulx.localization.js:10]: you should not be changing that as its the namespace that we use

File: public/js/vulx.localization.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const loadPath = `/locales/{{lng}}/{{ns}}.json`;
+const loadPath = `/locales/{{lng}}/{{ns}}.lang`;
```
Comment (ItsSyfe) [inline public/js/vulx.localization.js:1]: please do not change it from using a json to a lang as its not the correct format for i18next

**Review by ItsSyfe — CHANGES_REQUESTED** [review]
You shouldn't be changing these as you're deviating from the standards set out by i18next, and we'd like you to review this and revert these changes.

**Comment by ItsSyfe** [comment]
Also as for commits we'd like you to conform to [the conventional commits format.](https://www.conventionalcommits.org/)

**Comment by NeXiDE** [comment]
I have added a proper support for translation & changing languages via the dashboard into a different branch and made a pull request to the branch of @YeetHackz 

@ItsSyfe . There are some issues that I can't fix though that are minor. Could you review them and make changes to it?
[Link to the Pull Request](https://github.com/YeetHackz/Vulx-i18n/pull/1)

**Review by AquaPlaysYT — APPROVED** [review]
Ready for squash & merge.
